OriginateTheme/Scripts/ot_reference_resolver_test.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
+    otgenerator.py
+    Collection of tests for the ot_generator.py script.
+"""
+
+import os
+import unittest
+import copy
+import pprint
+
+from ot_reference_resolver import *
+
+pp = pprint.PrettyPrinter(indent=4)
+
+class ResolveComponentReferencesTestCase(unittest.TestCase):
+    def runTest(self):
+        data = json.loads("""{
```

Comment:
Since this json is the same as in the following test case extract it and use it in both Test Cases.